src/pages/admin/volunteers/[volId]/index.tsx

```diff
@@ -20,13 +21,20 @@ interface Props {
 
 const VolunteerPage: NextPage<Props> = ({ vol }) => {
     const classes = useStyles();
+    const router = useRouter();
 
     if (!vol) {
         return <Error statusCode={404} />;
     }
 
     const firstName = vol.name.split(" ")[0];
 
+    const handleEditClick = async () => {
+        if (vol._id) {
+            await router.push(`/admin/volunteers/${vol._id}/update`);
```

Comment:
Could you add this route to `utils/urls.ts` under `pages`? And then use that here? So you'd add:
```updateVolunteer: (volId: string) => `/api/events/${volId}`,```
then you can use that here with `router.push(urls.pages.updateVolunteer(volId)`. 

The reason I like it this way is in case we change the routes and to have all the pages of the site in one place that we can reference.